src/db/gc/mod.rs

```diff
@@ -167,15 +169,32 @@ impl<DB: Blockstore + GarbageCollectable + Sync + Send + 'static> MarkAndSweep<D
         }
     }
 
+    fn update_last_gc_run(&self, epoch: ChainEpoch) -> anyhow::Result<()> {
+        self.db
+            .write_bin(SETTINGS_KEY, epoch.to_string().as_bytes())
+    }
+
+    // Unfortunately there seems to be no good way of decoding a slice into i64 without array init
+    // and manipulation, therefore a string representation is used.
```

Comment:
Are you sure?
```
i64::from_le_bytes(self.db.read_bin(SETTINGS_KEY)?.unwrap_or_default().try_into()?)
```

Reply:
Depends on what you consider a `good way`. I'm aware of the `try_into` option.